opslevel/resource_opslevel_service.go

```diff
@@ -234,7 +234,7 @@ func resourceServiceCreate(d *schema.ResourceData, client *opslevel.Client) erro
 		}
 		_, err := client.ServiceApiDocSettingsUpdate(string(resource.Id), docPath.(string), source)
 		if err != nil {
-			log.Error().Err(err).Msgf("failed to update service '%s' api doc settings", resource.ManagedAliases[0])
+			log.Error().Err(err).Msgf("failed to update service '%s' api doc settings", resource.Aliases[0])
```

Comment:
Noticed that this is a bug that was introduced in services. If the user has no set aliases, this line will crash instead of erroring. `Aliases[0]` is being used because it's guaranteed that the slug/friendly_id will be present there.

Reply:
❤️